runtime/verutil/cfrerr.c

```diff
@@ -48,15 +48,53 @@ getJ9CfrErrorDetailMessageNoMethod(J9PortLibrary* portLib, J9CfrError* error, co
 	/* J9NLS_CFR_ERROR_TEMPLATE_NO_METHOD=%1$s; class=%3$.*2$s, offset=%4$u */
 	template = j9nls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE, J9NLS_CFR_ERROR_TEMPLATE_NO_METHOD, "%s;%.*s,%u");
 
-	allocSize = strlen(template) + strlen(errorDescription) + MAX_INT_SIZE + classNameLength;
+	allocSize = strlen(template) + strlen(errorDescription) + classNameLength + MAX_INT_SIZE;
 	errorString = j9mem_allocate_memory(allocSize, OMRMEM_CATEGORY_VM);
-	if (errorString != NULL) {
+	if (NULL != errorString) {
 		j9str_printf(PORTLIB, errorString, allocSize, template, errorDescription, classNameLength, className, error->errorOffset);
 	}
 
 	return errorString;
 }
 
+const char*
+getJ9CfrErrorBsmMessage(J9PortLibrary* portLib, J9CfrError* error, const U_8* className, UDATA classNameLength)
+{
+	PORT_ACCESS_FROM_PORT(portLib);
+	UDATA allocSize = 0;
+	char *errorString = NULL;
+	const char *template;
```

Comment:
Either assign `j9nls_lookup_message()`'s return value to this variable directly or initialize it to null

Reply:
Already fixed as suggested.